priv/migrations/pgsql_4.2.0_5.0.0.sql

```diff
@@ -0,0 +1,143 @@
+-- MOD_LAST
+
+ALTER TABLE last ADD COLUMN server varchar(250);
+UPDATE last SET server = 'localhost';
+ALTER TABLE last ALTER COLUMN server SET NOT NULL;
+
+CREATE UNIQUE INDEX i_last_su ON last (server, username);
+CREATE INDEX i_last_server_seconds ON last (server, seconds);
+DROP INDEX i_last_seconds;
+
+BEGIN;
+ALTER TABLE last DROP CONSTRAINT last_pkey;
+ALTER TABLE last ADD  CONSTRAINT last_pkey PRIMARY KEY USING INDEX i_last_su;
+COMMIT;
+
+-- MOD_PRIVACY
+
+-- Table privacy_default_list
+ALTER TABLE privacy_default_list ADD COLUMN server varchar(250);
+UPDATE privacy_default_list SET server = 'localhost';
+ALTER TABLE privacy_default_list ALTER COLUMN server SET NOT NULL;
+
+CREATE UNIQUE INDEX i_privacy_default_list_su ON privacy_default_list (server, username);
+
+BEGIN;
+ALTER TABLE privacy_default_list DROP CONSTRAINT privacy_default_list_pkey;
+ALTER TABLE privacy_default_list ADD CONSTRAINT privacy_default_list_pkey PRIMARY KEY USING INDEX i_privacy_default_list_su;
```

Comment:
Could you investigate possible simplification of the queries and avoiding the extra index? For example, I think this should work:
 
```
alter table privacy_default_list drop constraint privacy_default_list_pkey, add primary key (server, username);
```